### PR TITLE
Update l-te_sv.ts

### PR DESCRIPTION
**Pull request by bittin**
Updated Swedish translation

### CODE REVIEW

**Review by beanpole135 — CHANGES_REQUESTED** [review]
All of these translations are in the wrong spot and will not be used they way they are now.
The `type="unfinished"` tag needs to be removed, and the translated text put between the tags like this:
`<translation>This is translated text</translation>`

**Review by beanpole135 — CHANGES_REQUESTED** [review]
Looks like quoting and `<` errors in here still.
I suggested fixes for the first couple to get you started, but you will need to fix the rest too.

**Comment by bittin** [comment]
fixed

File: src-qt5/desktop-utils/lumina-textedit/i18n/l-te_sv.ts [hunk 1]
```diff
@@ -312,15 +312,16 @@ Vill du stänga den ändå ?
...
     </message>
     <message>
         <location filename="../MainUI.cpp" line="547"/>
         <source>There are unsaved changes.
 Do you want save them before you close the editor?
 
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Det finns osparade ändringar.
+                           Vill du spara dom innan du stänger editorn?</translation>
```
Comment (beanpole135) [inline src-qt5/desktop-utils/lumina-textedit/i18n/l-te_sv.ts:324]:
```suggestion
                           Vill du spara dom innan du stänger editorn?
                           
                           %1</translation>
```
Ensure the replaceable-string tag is added back in.

**Review by beanpole135 — APPROVED** [review]

**Comment by beanpole135** [comment]
Thanks for fixing that!

**Pull request merged**